<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KVirtualIfaceWidget</class>
 <widget class="QWidget" name="KVirtualIfaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_model">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_model">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">virtio</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">ne2k_pci</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">i82551</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">i82557b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">i82559er</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">rtl8139</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">e1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">pcnet</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Ignored</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>37</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLabel" name="label_connect">
     <property name="text">
      <string>Connected to</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="comboBox_type">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Nothing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">tap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">vde</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="lineEdit_file">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">tap10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="label_mac">
     <property name="text">
      <string>MAC Adress</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QLineEdit" name="lineEdit_mac">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="inputMask">
      <string notr="true">HH:HH:HH:HH:HH:HH; </string>
     </property>
     <property name="text">
      <string notr="true">52:54:00:12:34:56</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="pushButton_hwaddr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>49</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5" rowspan="2" colspan="3">
    <widget class="KUrlRequester" name="kurlrequester_scriptdown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">/etc/kvm/kvm-ifdown</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="checkBox_scriptdown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Script (down)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>300</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="checkBox_scriptup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Script (up)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="3">
    <widget class="KUrlRequester" name="kurlrequester_scriptup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">/etc/kvm/kvm-ifup</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
